backend/src/types/env.ts

```diff
@@ -21,6 +21,9 @@ export interface Env {
 	INSTANCE_DESCR: string
 	VAPID_JWK: string
 	DOMAIN: string
+	INSTANCE_ACCEPTING_REGISTRATIONS?: boolean
+	INSTANCE_REGISTRATIONS_REQUIRE_APPROVAL?: boolean
+	INSTANCE_CONFIG_STATUSES_MAX_CHARACTERS?: number
```

Comment:
Not sure INSTANCE_CONFIG_STATUSES_MAX_CHARACTERS is useful because we don't respect it yet. It's hardcoded to 500 (or some value) at the moment.

Reply:
Agreed. When I wrote this code, the work on the Admin component hadn't been merged to `main` so all of these `ENV` vars I added are a moot point. I'll remove all of them for now. `INSTANCE_CONFIG_STATUSES_MAX_CHARACTERS` may find its way back in [PR #366](https://github.com/cloudflare/wildebeest/pull/366) because that PR deals with several validation methods for ActivityPub and Mastodon objects